shared/video-sdk/develop/real-time-transcription/project-implementation/flutter.mdx

```diff
@@ -48,13 +48,13 @@ You see errors in your IDE. This is because these widgets refer to variables and
     String apiSecret = "<your API secret from Agora console>";
     String instanceId = "Test-1";
     String authorizationHeader = "", builderToken = "";
-    String speechToTextTaskId = ""; // Returned by the start method
-    bool isSpeechToTextRunning = false;
+    String realTimeTranscriptionTaskId = ""; // Returned by the start method
+    bool isRealTimeTranscriptionRunning = false;
     ```
 
-### Manage a Real-Time Transcription task
+### Manage a real-time transcription task
 
-Take the following steps to start, stop and query a speech-to-text task in your <Vpl k="CLIENT" />.
+Take the following steps to start, stop, and query a real-time transcription task in your <Vpl k="CLIENT" />.
```

Comment:
`stop,` 
Please remove ,(comma)

Reply:
This is Oxford comma, we are using it according to the style guide

Reply:
Actually for the same case, it's not there in other platforms.